tests/Feature/ContactTest.php

```diff
@@ -212,6 +212,26 @@ public function test_a_contact_can_have_food_preferences()
         $this->assertDatabaseHas('contacts', $food);
     }
 
+    public function test_a_contact_can_have_its_last_name_removed()
+    {
+        list($user, $contact) = $this->fetchUser();
+
+        $data = [
+            'firstname' => $contact->first_name,
+            'lastname' => '',
+            'gender' => $contact->gender_id,
+            'birthdate' => 'unknown',
+        ];
+
+        $this->post('/people/'.$contact->id.'/update', $data);
```

Comment:
I think it should use `hashID` instead of `->id`

Reply:
Should I take the opportunity to update all other code (within the ContactTest class) so that it stays consistent?

Reply:
@tomzx Oh I didn't saw it's in a test. Well, I think it's not a bad idea to fix these tests, go ahead !

Reply:
@tomzx we will fix the contactid uses in test later...